src/Neon/Class/HasAdd.purs

```diff
@@ -18,5 +18,8 @@ instance listHasAdd :: HasAdd (Data.List a) where
 instance numberHasAdd :: HasAdd Number where
   add y x = Prelude.add x y
 
+instance orderingHasAdd :: HasAdd Data.Ordering where
+  add y x = Prelude.append x y
```

Comment:
Also, for what it's worth, the `Prelude` includes a `Semigroup` instance for `Ordering`. See <https://pursuit.purescript.org/packages/purescript-prelude/0.1.4/docs/Prelude#t:Ordering>.